interpretation/requirements.txt

```diff
@@ -1,3 +1,6 @@
 numpy>=1.17.0
 azure-cli-core
-azureml-sdk==1.0.74
\ No newline at end of file
+azureml-sdk==1.0.83
```

Comment:
how is this different from the dutchf3_patch/local requirements file?

Reply:
@maxkazmsft good question, @annazietlow since you made this modification can you provide Max with the rational here?

Reply:
@maxkazmsft I just reran this with azureml-sdk==1.0.74 and we had to update this because it wasn't compatible with other azureml libraries we were using.

![image](https://user-images.githubusercontent.com/8856246/74578382-33e3de80-4f49-11ea-9187-215805f39327.png)

Reply:
the dutchf3_patch file is used to create the environment for running the training step in AML. The changes here keep the repo up to date for running the kickoff_train file